R/summary.pop_data.R

```diff
@@ -28,57 +32,65 @@ summary.pop_data <- function(object, strata = NULL, ...) {
     object %>%
     distinct(across(all_of(cols)))
 
-  cat("\nn =", nrow(ages), "\n")
```

Comment:
I have moved the `cat()` calls to `print.summary.pop_data()` (below), to match standard practices.